index.md

```diff
@@ -8,29 +8,29 @@ active_tab: main_page
 ---
 
 
-Course number
+Course number:
```

Comment:
Doesn't this cause two `:`s to be added? There is one at the beginning of the next line.